tests/Unit/Bridge/Doctrine/DBAL/Types/TypesDumperTest.php

```diff
@@ -13,6 +13,9 @@
 use Elao\Enum\Bridge\Doctrine\DBAL\Types\TypesDumper;
 use Elao\Enum\Tests\TestCase;
 
+/**
+ * @requires OS Linux|Darwin
```

Comment:
The problem on Windows is that the fixtures are using default `\n` line endings, while generated files end up having `\r\n` on Windows causing file content assertions to fail. I believe all PHP files should be generated with `\n` line endings as described in [PSR-2](https://www.php-fig.org/psr/psr-2/#:~:text=2.2.%20Files).

This is because file templates stored in [heredoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) appear to get local system file endings.
One solution would be to keep the file templates in separate files, like it's done in  Symfony Maker Bundle templates: 
https://github.com/symfony/maker-bundle/blob/main/src/Resources/skeleton/controller/Controller.tpl.php

Reply:
I would suggest to merge this PR as it's green ✅  and I will try to fix the issue in a separate PR, branching off this changes because Windows builds are currently broken on `master`.